client/components/promo-section/promo-card/style.scss

```diff
@@ -1,3 +1,6 @@
+@import '~@wordpress/base-styles/breakpoints';
+@import '~@wordpress/base-styles/mixins';
```

Comment:
Had to look up this one: it's a sass loader import

https://github.com/webpack-contrib/sass-loader#resolving-import-at-rules